### PR TITLE
Pint 1842

### PR DESCRIPTION
**Pull request by jumpingGrendel**
### Proposed changes

Added documentation for Airship's Destination Action

### Merge timing
- ASAP once approved?

### CODE REVIEW

File: src/connections/destinations/catalog/actions-airship/index.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+1. From the Segment web app, click **Catalog**, then click **Destinations**.
+2. Find the Destinations Actions item in the left navigation, and click it.
+3. Click **Configure Airship (Actions)**.
+4. Select an existing Source to connect to Airship (Actions).
+5. Name the destination and choose between filling in the settings manually or copying from an existing instance.
+6. Click Create Destination
+7. Enter your Access Token and App Key - these can be optained by going to your Airship project and navigating to Settings->Partner Integrations and selecting Segment. Following the instructions there will create a Tag Group, Attributes, and provide the Access Token and App Key.
+8. Select the appropriate data center. 
+
```
Comment (joe-ayoub-segment) [inline src/connections/destinations/catalog/actions-airship/index.md:37]: We could add some lines to explain which Preset Mappings are available, etc.

**Comment by joe-ayoub-segment** [comment]
Hi @jumpingGrendel , 
Thanks for getting this PR raised! 

Our docs team will pick it up and review, however I did have a quick read and left a couple of comments. 

Cheers,
Joe

File: src/connections/destinations/catalog/actions-airship/index.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# 💥 Segment Partner Source Documentation Template
+
+## Template begins here...
+
+---
+
+## title: Airship (Actions) Destination
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/actions-airship/index.md:7]: Please update this to the following:

```
---
title: CleverTap (Actions) Destination
id: 6475c5c14f7db4914bcd512f
---
```

**Comment by stayseesong** [comment]
@jumpingGrendel @joe-ayoub-segment @tcgilbert I'm going to close this PR out as I created an editable version here: #4946

**Pull request closed without merging**